tests/assets/nbs-moved-scripts/pipeline.yaml

```diff
@@ -0,0 +1,25 @@
+meta:
```

Comment:
@edublancas can we use instead of this new asset in one of the existing one and inject on top of it?

Reply:
just to confirm: looks like this `nbs-moved-scripts` was created to represent the user case where the `source`s are nested, right? if that's the only reason, we can use `nb-nested` to avoid creating a new assets folder just for this test.

Reply:
Just changed the test so that it uses `nb-nested` instead!

Reply:
Perfect, I think if you already have the fixture and template, there's no need to reinvent the wheel.